Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM golang:1.5.1
+ENV GO15VENDOREXPERIMENT 1
+
+ADD . /go/src/github.com/intercom/dvara
+RUN go install github.com/intercom/dvara/cmd/dvara
+ENTRYPOINT /go/bin/dvara -addrs=$ADDRS -max_connections=1000 -max_per_client_connections=100 --port_start=17017
```

Comment:
port_start is the proxy port, where the clients will connect to. Assuming that mongodb is running on 17017, a different port should be taken.
Default is 6000, so maybe that one? ( I don't have any feeling regarding is 6000 or a different number of good one).

addrs variable is a comma separated one, including the port:
dvara --help
  -addrs string
    	comma separated list of mongo addresses (default "localhost:27017")